DEPS

```diff
@@ -45,7 +45,7 @@ vars = {
   # Dart is: https://github.com/dart-lang/sdk/blob/main/DEPS.
   # You can use //tools/dart/create_updated_flutter_deps.py to produce
   # updated revision list of existing dependencies.
-  'dart_revision': '18df0aae67183931b1b6d5d12cc04d89f7137919',
+  'dart_revision': '9ad08f1c951ae8b5ee0387306f450afaccf4eaba',
```

Comment:
nit: This requires updating the license hash

Reply:
done. TIL. Thanks a lot for explaining!